src/components/WalletAddress.js

```diff
@@ -81,7 +81,7 @@ class WalletAddress extends React.Component {
 
     if (hathorLib.wallet.isHardwareWallet()) {
       $('#ledgerAlert').modal('show');
-      ledger.checkAddress(hathorLib.transaction.intToBytes(this.props.lastSharedIndex, 4));
+      ledger.checkAddress(this.props.lastSharedIndex);
```

Comment:
You must update `checkAddress` docstring. It's still saying that receives a buffer.

Reply:
Done.